pkg/ocm/helpers.go

```diff
@@ -273,3 +273,16 @@ func RandomLabel(size int) string {
 	}
 	return string(chars)
 }
+
+func (c *Client) LinkAccountRole(accountID string, roleARN string) error {
+	labelBuilder, err := amsv1.NewLabel().Key("sts_user_role").Value(roleARN).Build()
+	if err != nil {
+		return err
+	}
+	_, err = c.ocm.AccountsMgmt().V1().Accounts().Account(accountID).Labels().Add().
+		Body(labelBuilder).Send()
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
I think we should support multiple role ARNs in this label, maybe fetch the existing value, and see if there is anything in it. If there is, then append a comma-separated list of role ARNs